habitat-lab/habitat/tasks/nav/nav.py

```diff
@@ -1197,8 +1258,8 @@ def step(
         task: EmbodiedTask,
         linear_velocity: float,
         angular_velocity: float,
+        camera_pitch_angular_velocity: float = 0.0,
         time_step: Optional[float] = None,
-        allow_sliding: Optional[bool] = None,
```

Comment:
Is the agent no longer allowed to slide? Is this a breaking change that needs to be documented?

Reply:
This change was done by @exhaustin. One thing to note is that the velocity action was mostly not in use in habitat 1.0, so the number of people affected by this would be minimal. But that doesnt mean that we should stop allowing this completely. It might be useful to improve training performance.

Reply:
If it's only related to continuous control we can remove it.